radis/lbl/loader.py

```diff
@@ -704,7 +704,7 @@ def fetch_databank(
         parfuncfmt="hapi",
         levels=None,
         levelsfmt="radis",
-        load_energies=False,
+        load_energies=True,
```

Comment:
This was @anandxkumar  change. Energy not loaded by default unldss we actually need a nonequilibrium calculation. Let's try to keep it like this

Reply:
Since I am now calculating the trans_moment in factory.py I had to set the load_energies parameter. Any other way energies could be loaded at runtime?

Reply:
If that's required it would fail for all species that are not ready for nonequilibrium (I.e, all but CO and CO2). Can you run some tests with OH for instance? 

As I say in another comment, calculation of the transition moment looks expensive, maybe it isnt the right strategy. 

Before pushing forward, maybe just focus on molar mass and abundance, which are easy to implement. Then let's discuss about Einstein coefficients / linestrengths. It may also depend on what's usually available in data search others than HITRAN / HITEMP. If Exomol or GEISA provide Einstein A rather than Linestengths it may be more interesting to keep A and drop the linestength. Should be discussed on Slack!

Reply:
Yes, sure !!

Reply:
I will revert to my molar_mass commit, start working from there, and once abundance work is done we can get back to this.

Reply:
Don't revert everything ; just create a new branch (e.g: `Einstein` ) from here to keep the preliminary work on Einstein coefficients; and reset your `df-refactors` to the molar_mass commit